doc/doc_utils/jumpstart_doc_utils.py

```diff
@@ -227,22 +229,22 @@ def create_marketplace_model_table():
         if model["model_id"] not in sdk_manifest_top_versions_for_models:
             sdk_manifest_top_versions_for_models[model["model_id"]] = model
         else:
-            if Version(
-                sdk_manifest_top_versions_for_models[model["model_id"]]["version"]
-            ) < Version(model["version"]):
+            if str(sdk_manifest_top_versions_for_models[model["model_id"]]["version"]) < str(
```

Comment:
why are we not casting to `Version` (is it cause of proprietary versions)? This'll cause `2.100.0` > `2.9.0` to fail, unless we pad all the versions with zeros.

Reply:
Yes it is, `Version("v1.0.3-v1")` throws, as they could have wild versioning system. This should get the top one in our metadata which is the latest one, with string comparison